{"uuid": "809e5860-4fc0-48b1-bf55-a081a84b43d4", "children": ["92268424-c7d2-4f88-950d-6e212ec9cbd5"], "befores": [{"name": "driver", "status": "broken", "statusDetails": {"message": "urllib3.exceptions.ProtocolError: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))\n", "trace": "  File \"/home/andrei/.cache/pypoetry/virtualenvs/devops-sandbox-XsxpJM04-py3.11/lib/python3.11/site-packages/pluggy/_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/andrei/.cache/pypoetry/virtualenvs/devops-sandbox-XsxpJM04-py3.11/lib/python3.11/site-packages/_pytest/fixtures.py\", line 1130, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/andrei/.cache/pypoetry/virtualenvs/devops-sandbox-XsxpJM04-py3.11/lib/python3.11/site-packages/_pytest/fixtures.py\", line 902, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"/home/andrei/Desktop/code/devops-sandbox/conftest.py\", line 26, in driver\n    driver = webdriver.Remote(\n             ^^^^^^^^^^^^^^^^^\n  File \"/home/andrei/.cache/pypoetry/virtualenvs/devops-sandbox-XsxpJM04-py3.11/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 286, in __init__\n    self.start_session(capabilities, browser_profile)\n  File \"/home/andrei/.cache/pypoetry/virtualenvs/devops-sandbox-XsxpJM04-py3.11/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 378, in start_session\n    response = self.execute(Command.NEW_SESSION, parameters)\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/andrei/.cache/pypoetry/virtualenvs/devops-sandbox-XsxpJM04-py3.11/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 438, in execute\n    response = self.command_executor.execute(driver_command, params)\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/andrei/.cache/pypoetry/virtualenvs/devops-sandbox-XsxpJM04-py3.11/lib/python3.11/site-packages/selenium/webdriver/remote/remote_connection.py\", line 290, in execute\n    return self._request(command_info[0], url, body=data)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/andrei/.cache/pypoetry/virtualenvs/devops-sandbox-XsxpJM04-py3.11/lib/python3.11/site-packages/selenium/webdriver/remote/remote_connection.py\", line 311, in _request\n    response = self._conn.request(method, url, body=body, headers=headers)\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/andrei/.cache/pypoetry/virtualenvs/devops-sandbox-XsxpJM04-py3.11/lib/python3.11/site-packages/urllib3/_request_methods.py\", line 118, in request\n    return self.request_encode_body(\n           ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/andrei/.cache/pypoetry/virtualenvs/devops-sandbox-XsxpJM04-py3.11/lib/python3.11/site-packages/urllib3/_request_methods.py\", line 217, in request_encode_body\n    return self.urlopen(method, url, **extra_kw)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/andrei/.cache/pypoetry/virtualenvs/devops-sandbox-XsxpJM04-py3.11/lib/python3.11/site-packages/urllib3/poolmanager.py\", line 433, in urlopen\n    response = conn.urlopen(method, u.request_uri, **kw)\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/andrei/.cache/pypoetry/virtualenvs/devops-sandbox-XsxpJM04-py3.11/lib/python3.11/site-packages/urllib3/connectionpool.py\", line 844, in urlopen\n    retries = retries.increment(\n              ^^^^^^^^^^^^^^^^^^\n  File \"/home/andrei/.cache/pypoetry/virtualenvs/devops-sandbox-XsxpJM04-py3.11/lib/python3.11/site-packages/urllib3/util/retry.py\", line 470, in increment\n    raise reraise(type(error), error, _stacktrace)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/andrei/.cache/pypoetry/virtualenvs/devops-sandbox-XsxpJM04-py3.11/lib/python3.11/site-packages/urllib3/util/util.py\", line 38, in reraise\n    raise value.with_traceback(tb)\n  File \"/home/andrei/.cache/pypoetry/virtualenvs/devops-sandbox-XsxpJM04-py3.11/lib/python3.11/site-packages/urllib3/connectionpool.py\", line 790, in urlopen\n    response = self._make_request(\n               ^^^^^^^^^^^^^^^^^^^\n  File \"/home/andrei/.cache/pypoetry/virtualenvs/devops-sandbox-XsxpJM04-py3.11/lib/python3.11/site-packages/urllib3/connectionpool.py\", line 536, in _make_request\n    response = conn.getresponse()\n               ^^^^^^^^^^^^^^^^^^\n  File \"/home/andrei/.cache/pypoetry/virtualenvs/devops-sandbox-XsxpJM04-py3.11/lib/python3.11/site-packages/urllib3/connection.py\", line 454, in getresponse\n    httplib_response = super().getresponse()\n                       ^^^^^^^^^^^^^^^^^^^^^\n  File \"/usr/lib64/python3.11/http/client.py\", line 1375, in getresponse\n    response.begin()\n  File \"/usr/lib64/python3.11/http/client.py\", line 318, in begin\n    version, status, reason = self._read_status()\n                              ^^^^^^^^^^^^^^^^^^^\n  File \"/usr/lib64/python3.11/http/client.py\", line 287, in _read_status\n    raise RemoteDisconnected(\"Remote end closed connection without\"\n"}, "start": 1684920210462, "stop": 1684920223833}], "start": 1684920210462, "stop": 1684920224079}